code/index.html

```diff
@@ -1,30 +1,120 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
-  </head>
-  <body>
-    <!-- video or image as a header is cool :) -->
 
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>The Breakfast Club</title>
+  <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,700,700i|Source+Sans+Pro" rel="stylesheet">
+</head>
+
+<body>
+
+  <!-- Image header -->
+
+<div class="hero-img">
+  <h1>The Breakfast Club</h1>
+</div>  
+
+
+  <!-- Who we are text -->
+  <div class="weare">
+    <h2> "We are a feel-good company that wants to create a community
+      for breakfast lovers and small businesses" </h2>
+  </div>
+
+  <!-- Sign up text-->
+
+  <div class="sign-up">
+    <h1>Sign up for our weekly meet-ups! </h1>
+  </div>
+
+
+  <section class="container">
     <!-- Signup form -->
 
+    <form action="http://httpbin.org/anything" method="POST">
+      <div class="form-group">
+        <label>
+          Name
+          <input type="text" name="name" required />
+        </label>
+      </div>
+
+      <div class="form-group">
+        <label>
+          Phone
+          <input type="tel" name="phone" required />
+        </label>
+      </div>
+
+      <div class="form-group">
+        <label>
+          Email
+          <input type="email" name="email" required />
+        </label>
+      </div>
+
+      <h3>City</h3>
+
+      <div class="form-group">
+        <label>
+          <input type="radio" name="city" id="Stockholm" value="sthlm" />
+          Stockholm
+        </label>
+        <label>
+          <input type="radio" name="city" id="Gothenburg" value="gbg" />
+          Gothenburg
+        </label>
+      </div>
+
+      <div class="form-group">
+        <label>
+          <input type="checkbox" name="checkbox" id="check" />
+          I Agree to the terms and conditions
```

Comment:
Maybe you'd want this required before submitting? Otherwise it doesn't fill any function